Formula/arkade.rb

```diff
@@ -32,9 +32,7 @@ def install
 
     bin.install_symlink "arkade" => "ark"
 
-    (zsh_completion/"_arkade").write Utils.safe_popen_read(bin/"arkade", "completion", "zsh")
-    (bash_completion/"arkade").write Utils.safe_popen_read(bin/"arkade", "completion", "bash")
-    (fish_completion/"arkade.fish").write Utils.safe_popen_read(bin/"arkade", "completion", "fish")
+    generate_completions
```

Comment:
I feel like `bin/"arkade"`, maybe even including `completion` should be a mandatory argument because otherwise this DSL doesn't make it obvious that it's going to be shelling out to three commands. Alternatively, `generate_completions_from_executable` or something might make this more obvious.